code/modules/projectiles/projectile/bullets.dm

```diff
@@ -12,6 +12,14 @@
 	damage = 5
 	stamina = 40
 
+/obj/item/projectile/bullet/weakbullet/on_hit(atom/target, blocked = 0)
+	. = ..()
+	if(isliving(target))
+		var/mob/living/M = target
+		if(M.move_resist < INFINITY)
+			var/atom/throw_target = get_edge_target_turf(M, get_dir(src, get_step_away(M, src)))
```

Comment:
This may be a better solution for consistency, maybe swapping firer and M if needed
```suggestion
			var/atom/throw_target = get_edge_target_turf(M, get_dir(src, get_step_away(firer, M)))
```